README.md

```diff
@@ -187,6 +187,12 @@ _Note: You can only connect to servers running inside docker-compose. If you nee
  5. `yarn run ci:wait && yarn run cypress:open`
  6. Start adding tests, they get rerun automatically when open!
 
+### Using GUI in Docker
+ 1. If you are on Mac, install the XQuartz X11 server (`brew cask install xquartz`) and restart your machine
+ 2. Run `xhost local:root` to allow the root user from the Docker container to send messages to the X server
+ 3. `docker-compose up -d cypressserver` and wait until the seeds have finished (can take ~5 minutes, you can check using `docker-compose logs cypressserver`)
+ 4. `docker-compose run cypress-gui`
```

Comment:
Warum nicht einfach `docker-compose run cypress-gui` gibt ja extra den entrypoint im cypress, der wartet, bis hitobito bereit ist…

Reply:
+1 für das, aber für neue User ist es eventuell unverständlich, warum das so lange dauert. Man könnte bei 4. darauf hinweisen, dass 3. dadurch auch ausgelöst würde?

Reply:
Klar, würde die Message in Klammern beibehalten. Könnte man allenfalls gleich auch im [entrypoint](https://github.com/cleverer/hitobito-docker/blob/e82d0d9755c7a784d4948cf8576dbbc46ca6da21/.docker/cypress/entrypoint.sh#L5) hinterlegen.